site/layouts/partials/ocw_news.html

```diff
@@ -0,0 +1,21 @@
+<div class="ocw-news mx-auto container">
+    <h3><span class="ocw">OCW</span> News</h3>
+    <div class="d-grid items-row">
+        {{- $newsItems := getJSON (print (strings.TrimSuffix "/" (getenv "OCW_STUDIO_BASE_URL")) "/api/news/") -}}
```

Comment:
if the api call fails or there is 0 entries,  we probably dont need the title / and "more" button there.  Also see https://github.com/mitodl/ocw-www/issues/22#issuecomment-761108310

Reply:
getJSON will break the build if the URL does not resolve. I don't think it will ever be true that the news feed returns an empty list since that would only happen if all news items were deleted from ocw-openmatters.org while the web site was still functioning

Reply:
ok, It could be other reasons,  like a bug in your code in ocw-studio.  I thought you could just move this line on top.  Just a suggestion, up to you. It was a bit embarrassing for me to show https://ocwnext.odl.mit.edu/ in the presentation yesterday. (new courses,  previews /next buttons )